src/themes/legacy.scss

```diff
@@ -12,3 +12,19 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-banner.my-preview { display: none; }
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-old-icon { display: block; }
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-icon { display: none; }
+// temporary logic for legacy
```

Comment:
this comment needs to be added on line 11, the rules above belong to the same category (temporary logic)